config/spotbugs-exclude.xml

```diff
@@ -175,4 +175,13 @@
         <Method name="processFiltered"/>
         <Bug pattern="NP_GUARANTEED_DEREF_ON_EXCEPTION_PATH"/>
     </Match>
+    <Match>
+        <!-- Suppress the bug report of SuppressionXpathSingleFilter.xpathFilter not
+                initialized in constructor and dereferenced in com.puppycrawl.
+                tools.checkstyle.filters.SuppressionXpathSingleFilter.
+                accept(TreeWalkerAuditEvent) -->
```

Comment:
same.

Something like can't move field since we won't have the values to create the instance until the setters are called.